galaxy_ng/app/signals/handlers.py

```diff
@@ -48,6 +50,26 @@ def create_namespace_if_not_present(sender, instance, created, **kwargs):
     Namespace.objects.get_or_create(name=instance.namespace)
 
 
+def set_role_definitions(sender, **kwargs):
+    apps = kwargs.get("apps")
+    if apps is None:
+        from django.apps import apps
+    role_prefix = f"{sender.label}."
+    desired_roles = {}
+    for viewset in GALAXY_VIEWSETS:
+        locked_roles = GALAXY_VIEWSETS[viewset]['LOCKED_ROLES']
+        if locked_roles is not None:
+            desired_roles.update(locked_roles or {})
+    adjust_roles(apps, role_prefix, desired_roles, 0)
```

Comment:
I would use either of these (or if you really _want_ to shut it up pass it as kwarg `verbosity=0`.)
```suggestion
    adjust_roles(apps, role_prefix, desired_roles)
```
```suggestion
    adjust_roles(apps, role_prefix, desired_roles, verbosity=kwargs.get("verbosity", 0))
```

Reply:
Thanks for catching this.  I meant to correct it before opening the PR...